customersupport/static/list-tickets.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+    
+    $("#listTicketsButton").click(function(){
+        $.get("api/ticket/customer",{customer_id:$("#customer").val()})
```

Comment:
I was going to ask you what you wanted me to do about that one. I think disabling it would be a good option.